lib/core/services/identity-user.service.ts

```diff
@@ -22,17 +22,33 @@ import { AppConfigService } from '../app-config/app-config.service';
 import { IdentityGroupModel } from '../models/identity-group.model';
 import { IdentityRoleModel } from '../models/identity-role.model';
 import { IdentityUserModel } from '../models/identity-user.model';
-import { IdentityJoinGroupRequestModel, IdentityUserServiceInterface, IdentityUserPasswordModel, IdentityUserQueryCloudRequestModel, IdentityUserQueryResponse } from './identity-user.service.interface';
-import { JwtHelperService } from './jwt-helper.service';
+import {
+    IdentityJoinGroupRequestModel,
+    IdentityUserServiceInterface,
+    IdentityUserPasswordModel,
+    IdentityUserQueryCloudRequestModel,
+    IdentityUserQueryResponse
+} from './identity-user.service.interface';
 import { OAuth2Service } from './oauth2.service';
 
+interface UserInfoInterface {
```

Comment:
the Interface word in the end is not needed, UserInfo should be enough

Reply:
interface word removed